docs/04-documentation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: How to update documentation on ECL
+label: Edit documentation
+---
+
+This page explains how to update components documentation directly in GitHub
+
+## Step 0: GitHub account
+
+If you don't already have a GitHub account, make sure to create one and log on with it.
+
+## Step 1: Open documentation file
+
+* Go to [ECL repository](https://github.com/ec-europa/europa-component-library)
```

Comment:
It would be useful to have visual hints/screenshots. You can check [INNO-540](https://webgate.ec.europa.eu/CITnet/jira/browse/INNO-540) for the initial suggestions.